backend/apis/clients/integrations/jpls-integration.js

```diff
@@ -1,6 +1,18 @@
-module.exports = function juntosPorLaSalud(integration) {
-  // TO-DO actually implement this
-  return Promise.resolve({
-    error: null
-  });
+const { upsertClient } = require("juntos-por-la-salud-node-client");
+
+module.exports = function juntosPorLaSalud(integration, client) {
+  return upsertClient({
+    client,
+    participantId: integration.externalId,
+    username: process.env.JPLS_USERNAME,
```

Comment:
With the new .env info do we all need to update ours or is it only relevant if we are working on that section?

Reply:
Only if we are working on that section

Reply:
Only if we are working on that section